marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/job/DeleteJobsTest.java

```diff
@@ -0,0 +1,129 @@
+package com.marklogic.hub.dataservices.job;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.FailedRequestException;
+import com.marklogic.client.dataservices.ExecCaller;
+import com.marklogic.client.dataservices.IOEndpoint;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.dataservices.JobService;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.step.RunStepResponse;
+import com.marklogic.hub.test.Customer;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DeleteJobsTest extends AbstractHubCoreTest {
```

Comment:
I don't see any need for this test - we don't expect anyone to use the DS directly, we expect clients to all use JobDataManager. So we only need to test that class, with that class handling all of the DS plumbing for us. Also avoids duplicating things like the error listener below.

Reply:
I think this test should still be merged with JobDataManagerTest - I would call the merged test "DeleteJobsTest". I don't think there's any value in having all this duplicated Bulk IO plumbing here, as JobDataManager takes care of all that.